### PR TITLE
[DOCS-6336] ADW 2.3

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
Added new branch for docs.
Added small section about new search.
Added new image.

### CODE REVIEW

**Review by mjstanford — CHANGES_REQUESTED** [review]
As far as I can tell there are only two sections with changes that require review (config/index.md and index.md) Correct?

Can't see any sign  of a screenshot in the search configuration section. Am I missing something?

For index.md please use "forms that can show specific filters based on your metadata" rather than "forms that can show specific information based on your metadata"

**Review by mjstanford — APPROVED** [review]

File: digital-workspace/2.2/install/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Installing Digital Workspace
+---
+There are a number of different ways to deploy Alfresco Digital Workspace. You can deploy it using Docker images that are packaged in Helm charts or using Docker Compose. You can also install Alfresco Content Services using standard WAR files contained in the distribution zip, and then configure the installation to include Alfresco Digital Workspace.
+
+The deployment methods are:
+
+* [Containerized deployment]({% link digital-workspace/latest/install/containerized.md %}). Due to the limited capabilities of Docker Compose, this deployment method is recommended for development and test environments only.
+* [Installing into Tomcat]({% link digital-workspace/latest/install/tomcat.md %})
+* [Installing into a different web server]({% link digital-workspace/latest/install/other-webserver.md %})
```
Comment (anxumalo) [inline digital-workspace/2.2/install/index.md:10]: In all the ADW 2.2 files, the internal links go to `/latest/` - multiple files need fixing.

**Review by anxumalo — CHANGES_REQUESTED** [review]
See review comments inline. 

Are you planning to fix the redirect that's mentioned in the JIRA ticket (DOCS-6336) in this PR as well?

File: digital-workspace/2.2/troubleshooting/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Yes, Alfresco Digital Workspace supports Alfresco Governance Services. See [Governance Services]({% link digital-workspace/latest/governance/index.md %}).
+
+### Does Alfresco Digital Workspace support Smart Folders
+
+Smart Folder access is supported but Smart Folders cannot be created in this application.
+
+### How do I setup SSL
+
+SSL configurations differ from one installation to the next. Here is one approach to the setup of NGINX using SSL, see [https://nginx.org/en/docs/http/configuring\_https\_servers.html](https://nginx.org/en/docs/http/configuring_https_servers.html){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply digital-workspace/2.2/troubleshooting/index.md]: fixed

File: digital-workspace/2.2/install/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply digital-workspace/2.2/install/index.md]: fixed

File: digital-workspace/2.2/config/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| viewer.maxRetries | The preview mechanism used to view a file tries to open it and if its unsuccessful it waits 20 seconds and then tries again. The process involves converting the file (multiple file formats are supported depending on content type) and if it takes more than 20 seconds the system will say that the content isn't supported. This time out can be increased by adding a value for this property. If you enter 2 the wait time before time out will be 40 seconds|
+
+## Permissions
+
+Permissions for folders and files in the Digital Workspace can be assigned and viewed easily.
+To view the permissions of a folder or file in the Digital Workspace, select the file or folder, right click it, and select **Permissions**. You are presented with the Permissions tab.
+
+The type of permissions a user or group has for a file is determined by what role they have.
+
+For more information on roles see [User roles and permissions](https://docs.alfresco.com/content-services/latest/using/permissions/).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply digital-workspace/2.2/config/index.md]: fixed.

**Review by anxumalo — COMMENTED** [review]
Missing update in `redirects/adw/concepts/configuration.html`

**Review by anxumalo — APPROVED** [review]

**Pull request merged**